src/services/donations/donations.hooks.js

```diff
@@ -117,15 +117,16 @@ const setUSDValue = async (context, donation) => {
     const { rate } = await getHourlyUSDCryptoConversion(context.app, createdAt, token.symbol);
 
     if (rate) {
+      const usVar = Number(
+        new BigNumber(amount)
+          .div(10 ** 18)
+          .times(rate)
+          .toFixed(5),
```

Comment:
Revert it to same `.toFixed(2)`, I didn't say to change it.

Reply:
Will revert to` .toFixed(2)`